com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -292,16 +292,13 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
             m_UseDrawProcedural = renderingData.cameraData.xr.enabled;
             m_UseFastSRGBLinearConversion = renderingData.postProcessingData.useFastSRGBLinearConversion;
 
+            var cmd = renderingData.commandBuffer;
             if (m_IsFinalPass)
             {
-                var cmd = CommandBufferPool.Get();
                 using (new ProfilingScope(cmd, m_ProfilingRenderFinalPostProcessing))
                 {
                     RenderFinalPass(cmd, ref renderingData);
                 }
-
-                context.ExecuteCommandBuffer(cmd);
-                CommandBufferPool.Release(cmd);
```

Comment:
Might be worth checking if there isn't anything in the context breaking command buffer execution in post. I guess not as at it should do is Blit / DrawProcedural, but I'm not 100% sure.